pegasus/helper_modules/hoc_event_review.rb

```diff
@@ -44,10 +44,12 @@ def self.events(**rest)
       select(:data, :secret, :processed_data).
       limit(100).
       map do |form|
+        next unless form[:processed_data]
         JSON.parse(form[:data]).
           merge(secret: form[:secret]).
           merge(JSON.parse(form[:processed_data]))
-      end
+      end.
+      compact
```

Comment:
Mostly learning from your PR more than reviewing :). Read the docs, but what does `compact` do here?

Reply:
the skipped rows (ones without processed data) will be mapped as 'nil', compact removes the nils from the returned result so we just have the data we care about

Reply:
Ah 🤦‍♂ -- misunderstood, was thinking the `next` already took care of the nulls. Got it, thanks!